src/doc/book/getting-started.md

```diff
@@ -24,41 +24,38 @@ see the website][platform-support].
 
 [platform-support]: https://forge.rust-lang.org/platform-support.html
 
-## Installing on Linux or Mac
+## Installing Rust
 
-If we're on Linux or a Mac, all we need to do is open a terminal and type this:
+All we need to do on On Unix systems like Linux and macOS is open a
```

Comment:
I know the tense didn't change with this PR, but maybe this "we" should be a "you'll"?